tests/validation/tests/v3_api/common.py

```diff
@@ -2483,3 +2483,17 @@ def wait_for_hpa_to_active(client, hpa, timeout=DEFAULT_TIMEOUT):
         assert len(hpas) == 1
         hpa = hpas[0]
     return hpa
+
+
+def get_node_details(cluster, client):
+    """
+    lists the nodes from the cluster. This cluster has only 1 node.
+    :return: client and node object
+    """
+    create_kubeconfig(cluster)
+    nodes = client.list_node(clusterId=cluster.id).data
+    assert len(nodes) > 0
+    for node in nodes:
+        if node.worker:
+            break
+    return client, node
```

Comment:
I guess this will error out since `node` is not defined out of the for loop.

Reply:
We will be able to access the value of node outside too.